src/app/api/services/searchService.js

```diff
@@ -1,42 +1,58 @@
-import pool from "./connection.js";
+  import pool from "./connection.js";
 
-const getTournamentsByName = async (name) => {
-  try {
-    const result = await pool.query(
-      `SELECT * FROM tournaments WHERE tournament_name ILIKE $1`,
-      [`%${name}%`],
-    );
+  const getTournamentsByName = async (name) => {
+    try {
+      const result = await pool.query(
+        `SELECT * FROM tournaments WHERE tournament_name ILIKE $1`,
+        [`%${name}%`],
+      );
 
-    if (result.rows.length === 0) {
-      return null;
+      if (result.rows.length === 0) {
+        return null;
+      }
+
+      return result.rows;
+    } catch (error) {
+      console.error("Error querying tournaments:", error);
+      throw new Error(`Database error: ${error.message}`);
+    }
+  };
+
+  const getGamesByName = async (name) => {
+    try {
+      const result = await pool.query(
+        `SELECT * FROM Games WHERE game_name ILIKE $1`,
+        [`%${name}%`],
+      );
+
+      if (result.rows.length === 0) {
+        return null;
+      }
+
+      return result.rows;
+    } catch (error) {
+      console.error("Error querying events:", error);
+      throw new Error(`Database error: ${error.message}`);
     }
+  };
 
-    return result.rows;
-  } catch (error) {
-    console.error("Error querying tournaments:", error);
-    throw new Error(`Database error: ${error.message}`);
-  }
-};
-
-const getGamesByName = async (name) => {
-  try {
-    const result = await pool.query(
-      `SELECT * FROM Games WHERE game_name ILIKE $1`,
-      [`%${name}%`],
-    );
-
-    if (result.rows.length === 0) {
-      return null;
+  const getAllTournaments = async () => {
```

Comment:
This function already exists in tournamentService, you could delegate to that if you want to (or improve it if you need).